pkg/azurefile/nodeserver.go

```diff
@@ -95,6 +99,41 @@ func (d *Driver) NodePublishVolume(ctx context.Context, req *csi.NodePublishVolu
 				return nil, status.Errorf(codes.InvalidArgument, fmt.Sprintf("invalid mountPermissions %s", perm))
 			}
 		}
+		if d.enableKataCCMount {
+			runtimeClass, err := getRuntimeClassForPodFunc(ctx, d.cloud.KubeClient, context[podNameField], context[podNamespaceField])
```

Comment:
`KubeClient` for the driver on the node may not have permission, this could make all volume mount fails no matter user is using katacc mount or not, so this should log a warning, and return as success

Reply:
it's possible that `getRuntimeClassForPod` return failure due to kubeclient not set or RuntimeClass not set in the pod, should return an error msg and then